src/ansys/aedt/core/generic/general_methods.py

```diff
@@ -1592,8 +1592,13 @@ def active_sessions(version=None, student_version=False, non_graphical=False):
                                 if i.pid == p.pid and (i.laddr.port > 50050 and i.laddr.port < 50200):
                                     return_dict[p.pid] = i.laddr.port
                                     break
-        except Exception:
-            pyaedt_logger.error("An error occurred while retrieving information for the active AEDT sessions")
+        except psutil.NoSuchProcess:  # pragma: no cover
+            # the process exited so it cannot be an active session
+            pass
```

Comment:
```suggestion
        except psutil.NoSuchProcess as e:  # pragma: no cover
            pyaedt_logger.error(
                f"The process exited and cannot be an active session: {e}"
            )
```

Reply:
@SMoraisAnsys could we instead issue a debug message here? This is not an error in my estimation: we are iterating over all the processes on the machine and one happened to exit between the time `psutil.process_iter()` returned it and (I suspect) we checked the name (or even more rarely, an `ansysedt` process happened to exit after we check the name but later in the loop).

Reply:
Fine by me !

Reply:
@isaacwaldron I updated the changes following the previous comment